src/test/java/testNG/TestJsonParser.java

```diff
@@ -0,0 +1,101 @@
+package testNG;
+
+import com.google.gson.JsonSyntaxException;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class TestJsonParser {
+    private Cart expectedCart;
+    private Cart actualCart;
+    JsonParser jsonParser;
+
+    @BeforeMethod
+    public void setParams() {
+        jsonParser = new JsonParser();
+    }
+
+    @Test (priority = 1)
+    public void testFileExists() {
+        expectedCart = new Cart("TestCart");
+
+        expectedCart.addRealItem(new RealItem());
+        expectedCart.addVirtualItem(new VirtualItem());
+
+        jsonParser.writeToFile(expectedCart);
+
+        File file = new File("src\\main\\resources\\TestCart.json");
```

Comment:
use `/` as separator. Java is system-agnostic language, the current `\\` is Windows-only